packages/web-config-server/src/apiV1/dataBuilders/generic/count/countMeasureValues.js

```diff
@@ -0,0 +1,64 @@
+import { DataBuilder } from '/apiV1/dataBuilders/DataBuilder';
+import { divideValues } from '/apiV1/dataBuilders/helpers';
+import { getMeasureBuilder } from '/apiV1/measureBuilders/getMeasureBuilder';
+
+class CountMeasureValuesBuilder extends DataBuilder {
+  async build() {
+    const {
+      measureBuilder,
+      convertToPercentage,
+      measureBuilderConfig,
+      valueKey = 'value',
+    } = this.config;
+
+    const { aggregator, dhisApi, query, entity } = this;
+
+    const { data: measureData } = await getMeasureBuilder(measureBuilder)(
```

Comment:
Just my personal opinion, my gut feeling is I don't quite prefer having a data builder that calls a measure builder like this 😢 . A reason I can think of is if we have a major refactor on measureBuilder class we might miss this one? 

If you agree, and if this is just to reuse the `getStringFromBinaryData` logic and count for `Multiple`, do you think you can reuse the `countCalculatedValuePerOrgUnit` data builder that I did for https://github.com/beyondessential/tupaia/pull/1429. (It has access to `calculateOperationForAnalytics.js` logic which can combine the string for you.

Reply:
Good idea. I don't necessarily mind using measure builders, but I agree that it would be cleaner to use the same dataBuilder for both.